plugins/lookup/kustomize.py

```diff
@@ -136,6 +139,9 @@ def run(
                 )
             )
 
+        if "build_flags" in kwargs:
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.